### ISSUE TITLE
Few enhancment suggesstions

### ISSUE THREAD

**Issue by Addonman** [RaiderIO/raiderio-addon#81]
Could be possiable to instead having main/alt or even better on top of it it have account score wher that is combination of chars progress not matter what char got best score in either raid or m+? I mean what happens if you have   raid best score on 1 char ,m+ score on alt-what score is show on 3rd alt?
Also would it be possiable to have 1 buton update for all you chars? Now this only posiable if they are in same guild.

**Comment by Addonman** [comment]
Some answer pls. 
Also is it possiable to sync chars in game-for example i update my char with app,log in game and if someone inspects me he gets newest update of my char.

Haveing score per account(best runs of all raids and m+) instead of oer best char would be immensly helpfull for ppl like my that have lot of chars on which we occcasionly do few m+ or 1-2 raids,but doesntt always have enough points to make it into your data update.

**Comment by tomrus88** [comment]
Agree. I've sent them same suggestion through the site feedback, but they never answered anything...

**Comment by jahraphael** [comment]
We don't currently have plans to do a combined "account score" in the addon. It's possible we might at some point show this on the website, but it's not currently on our near term plans.

We are alpha testing a new version of the addon with a number of changes. One of these makes it so that when we choose the "Main" to show for your M+ data, we find the scores for the *best* character on your account, rather than just the one you have marked as "Main". Currently this only applies to M+, but down the road we will apply the same type of logic to Raid Progress.

There's no one button update, but if you use the Raider.IO Desktop Client, it will automatically queue your characters for an update when you log out of them.

**Comment by jahraphael** [comment]
You can find more details about the alpha version of the addon by joining our Discord (https://discord.gg/raider) and looking at the pins in #addon-discussions.
